sdk/analytics/src/main/java/com/klaviyo/analytics/model/EventMetric.kt

```diff
@@ -14,5 +14,10 @@ sealed class EventMetric(name: String) : Keyword(name) {
     object ADDED_TO_CART : EventMetric("Added to Cart")
     object STARTED_CHECKOUT : EventMetric("Started Checkout")
 
-    class CUSTOM(name: String) : EventMetric(name)
+    open class CUSTOM(name: String) : EventMetric(name)
+
+    /**
+     * Internal Klaviyo event metrics are prefixed with a '$' character
+     */
+    internal val isInternal: Boolean = this.name.startsWith("$")
```

Comment:
Hmm, I think @belleklaviyo had a categorization of isGeofenceEvents and then && that with opened push. IDK what I lean towards, 
On one hand, yeah all these are internal but in a future would there be some internal events that diverge in their handling? I guess solving for now, I think I lean towards a categorization of isInternal since all internal events are handled the same rn. 

@belleklaviyo what do you think? Also, not a big deal to diverge between iOS and android but if it's no big deal might align for consistency.